src/store/itemShare.ts

```diff
@@ -42,9 +42,11 @@ export const useShareStore = defineStore("share", {
           })
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
         } catch (e: any) {
-          // create doc if it doesn't exist
           if (e.code === "not-found" && e.name === "FirebaseError")
-            await setDoc(docRef, { items: arrayUnion(item.text) })
+            try {
+              setDoc(docRef, { items: arrayUnion(item.text) })
+              // eslint-disable-next-line no-empty
+            } catch (err: unknown) {}
```

Comment:
not sure why you need the try/catch since you're not doing anything in the catch
you're also not `await` on the setDoc

Reply:
I have updated this code.
please refer -> https://i.imgur.com/kPUWcBx.png